homework_1/zad1/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"zad1/solution"
+)
+
+func main() {
+	startPtr := flag.Int("start",1,"First number")
+	endPtr := flag.Int("end", 2, "Last number")
```

Comment:
You're missing whitespaces after `,` in the first line, you've done it properly in the second line.

Regarding formatting of code in general, Golang has some conventions which are actually tied to the language. You can format your code by running `go fmt ./...` (it will format all `.go` files in this directory and all subdirectories). You can also configure automatic formatting of code in Goland by going to Preferences -> Tools -> File Watchers -> go fmt
However, I recommend not doing it automatically while writing the homework, but actually trying to do it correctly manually. Once you're done, you can validate how successful you were by actually running `go fmt` and then upload the homework.